platform/security/certificate/security-certificate-generator/src/main/java/org/codice/ddf/security/certificate/generator/CertificateCommand.java

```diff
@@ -160,14 +166,23 @@ public static String configureDemoCertWithDN(String[] dn) {
   public static String configureDemoCertWithDN(String[] dn, @Nullable String[] subjectAltNames) {
     CertificateSigningRequest csr = new CertificateSigningRequest();
     csr.setDistinguishedName(dn);
+
+    // Required for Chrome support (DDF-3104)
+    String commonName =
+        Arrays.stream(dn)
+            .filter(str -> str.toUpperCase().startsWith("CN="))
```

Comment:
✏️  It does not account for optional spaces around "CN"

Reply:
OBE per below comment.